admin/admin-base.php

```diff
@@ -302,6 +302,15 @@ public function admin_print_footer_scripts() {
 			$params = array_merge( $params, array( 'pll_term_id' => (int) $tag_ID ) );
 		}
 
+		/**
+		 * Filters the list of parameters to add to ajax request.
+		 *
+		 * @since 3.5
+		 *
+		 * @param array $params List of parameters.
+		 */
+		$params = apply_filters( 'pll_ajax_params', $params );
```

Comment:
If this filter is meant to be generic, maybe we should add a second parameter for the context. Maybe something like `'request prefilter'`? What do you think?

Reply:
Generic mean that we can put some other parameters not that we want to apply the filter elsewhere.
So, I think we can keep only one parameter.